sentences/cs/_common.yaml

```diff
@@ -331,26 +334,25 @@ expansion_rules:
   teplota: "{temperature}[ ][°|stupňů][ ][{temperature_unit}]"
   pozice: "{position}[[ ]%| procent]"
   volume: "{volume:volume_level}[[ ]%| procent]"
-
   brightness: "[na] {brightness}[%| procent]"
-  zavrit: "(zavři|zavřít)"
-  otevrit: "(otevři|otevřít)"
+  zavrit: "zavř(i|ít)"
+  otevrit: "otevř(i|ít)"
   roztahnout: "(roz|vy|od)táhn(out|i)"
-  zatahnout: "zatáhn(out|i)"
-  ktere: "(kter(á|é)|jak(é|á))"
-  prejit: "(přejít|přejdi)"
-  je_jsou: "(je|jsou)"
-  v: "(v|ve|na)"
-  y_ej: "(ý|ej)"
-  y_e: "(ý|é)"
+  zatahnout: "(za|s)táhn(out|i)"
+  prejit: "přej(ít|di)"
   koncentrace: "(úroveň|množství|koncentrace)"
-  jaky_je: "(jak<y_e> (je|bude)|jaká (je|bude)|jakou|jaká|jak|kolik|kolik (je|bude))"
+
+  y_e: "(ý|ej|á|é|ých|ejch)"
```

Comment:
I hope that this grouping of too many endings (which will produce lot of incorrect forms like dešťovéch srážek) won't increase error rate in detection.